ironic_prometheus_exporter/app/exporter.py

```diff
@@ -0,0 +1,30 @@
+import configparser
+import logging
+import os
+
+from flask import abort, Flask
+app = Flask(__name__)
+LOG = logging.getLogger(__name__)
+
+
+@app.route('/metrics', methods=['GET'])
+def prometheus_metrics():
+    try:
+        config = configparser.ConfigParser()
+        config.read(os.environ.get('IRONIC_CONFIG'))
+        DIR = config['oslo_messaging_notifications']['location']
+    except Exception as e:
+        LOG.error(e)
+        abort(404)
+
+    all_files = [os.path.join(DIR, name) for name in os.listdir(DIR)
+                 if os.path.isfile(os.path.join(DIR, name))]
+    content = ''
+    for file_name in all_files:
+        with open(file_name, 'r') as file:
```

Comment:
Probably not important on our scale.. but I'm not sure writing to files in python is atomic. You may end up reading half-written files. We can worry about it later, but let's keep it in mind.

Reply:
When the files are created we should write to files with different names and then rename the file (that operation is atomic).

Reply:
The oslo driver writes a file per node (`node_name`), so we should rename the files with some kind of prefix after written and Flask will only read files with the prefix?
like: `node1` we rename to `metrics_node1`, flask only read files in the directory that starts with `metrics_` ?

Reply:
Maybe write to a temporary directory and then move the files into the real directory?

Reply:
Ok, I'll update the driver to do this, since he is that write the files